features/support/ccd_support/et1_ccd_single_claimant.rb

```diff
@@ -136,11 +136,14 @@ def assert_acas_pdf_file
           expect(find_file_document(response, /\Aacas.*\.pdf\z/)).to be_present, 'Cannot find acas file - should start with acas and end in .pdf'
         end
 
-        def assert_acas_pdf_file_quantity(quantity)
-          count = response.dig('case_fields', 'documentCollection').count do |doc|
-            doc.dig('value', 'typeOfDocument') == 'ACAS Certificate'
+        def find_acas_names(quantity)
+          name = response.dig('case_fields', 'documentCollection')
+          acas_names = []
+          (1..5).each do |value|
+            acas_names << name[value]["value"]["uploadedDocument"]["document_filename"]
```

Comment:
You could try a map method similar to what i mentioned earlier rather than creating an array and gradually building it in a loop